cmd/deploy/endpoints.go

```diff
@@ -227,7 +227,7 @@ func (dc *EndpointGetter) showEndpoints(ctx context.Context, opts *EndpointsOpti
 		}
 	default:
 		if len(eps) == 0 {
-			oktetoLog.Printf("There are no available endpoints for '%s'\n", opts.Name)
+			fmt.Errorf(oktetoErrors.ErrEndpointsNotAvailable, opts.Name)
```

Comment:
Could you share a screenshot of how it looks like in the terminal?
This is not an error condition, it should be shown as an info message.

Reply:
Resolved!